gulpfile.js

```diff
@@ -1,10 +1,6 @@
 /* jshint node:true, camelcase:false */
 var gulp = require('gulp');
-var del = require('del');
-var glob = require('glob');
 var karma = require('karma').server;
-var merge = require('merge-stream');
```

Comment:
Can't remove this because it is still being used on the [analyze](https://github.com/CodeSeven/toastr/blob/2601964dc43eb573b2269641a5122dcce0cb6bc6/gulpfile.js#L30) task.